src/blueprints/tv2/entities/tv2-piece.ts

```diff
@@ -0,0 +1,6 @@
+import { PieceInterface } from '../../../model/entities/piece'
+import { Tv2PieceMetadata } from '../value-objects/tv2-metadata'
+
+export interface Tv2Piece extends PieceInterface {
```

Comment:
You are extending `PieceInterface`, so this should be `Tv2PieceInterface`.
Right now there are variables declared as `const pieceInterface: Tv2Piece` which leaves the question "why did they name it `PieceInterface` when it has the type of `Piece`?"

Reply:
I'll change the Tv2Piece type to be called Tv2PieceInterface, such that it matches, but pieceInterface is a bad variable name. It is not an interface, it is real data.

Reply:
It's the data needed to create an actual Piece entity.
I'm up for naming suggestions.

Reply:
Currently the "best" suggestion is `PieceData`.

Reply:
I support that suggested name. It is 'Data' needed to create a 'Piece'. It makes sense, atleast for me.

Reply:
I don't have any strong arguments for keeping the `xInterface` naming. Just remember to change it for `Rundown`, `Segment`, `Part` and `Piece` if you change it.